### DISCUSSION TITLE
Problem parsing PLOG reactions in FFCM2

### DISCUSSION THREAD

**Discussion by RSuryaNarayan** [AMReX-Combustion/PelePhysics#487] [Q&A] [answered]
Hello 

I am trying to parse [FFCM2](https://web.stanford.edu/group/haiwanglab/FFCM2/) into Pele and encounter an issue with PLOG reactions like so: 
![image](https://github.com/AMReX-Combustion/PelePhysics/assets/52670594/cb702898-df54-4ddb-bc92-65fa667a5aa2)

is there a way to modify cepter to bypass PLOG and treat them as normal reactions?

**Comment by marchdf** [comment, marked as answer]
could you modify the yaml input file so that it is not a plog? That would be the easiest.

**Comment by RSuryaNarayan** [comment]
thanks @marchdf how hard would it be to enable PLOG reactions in PeleC? Since we have large pressure variations in the system, I guess we want to retain it..

**Reply by marchdf** [reply]
we've talked about it and it would be great to add that capability... I think (?) it would be non-trivial. But idk much about plog so I am probably not the right person to ask.

**Reply by drummerdoc** [reply]
Full support of reactions in this form would be non-trivial because one would have to add support for them in the analytic Jacobian routine (then test it, etc).  Partial support would involve a much simpler augmentation of the parser to dump out the requisite evaluation formulas much like the other Arrhenius-type expressions that are already supported.  If only that was provided, you'd be able to use the time-explicit chemistry integrators, or the matrix-free implicit ones, which might be just fine for your application.  However, it would be best to add some sort of check that ensures that some other user doesn't *think* more code is there than really is.  Regardless, we are a bit light on staff right now that has the cycles to implement either of these additions for you, even though you are not the first to request it.  Happy to look over a PR that you might submit that could address these things though.

**Reply by baperry2** [reply]
This would be similar to running with SRK where we don't support methods that use the analytical Jacobian and raise errors if anything tries to compute the Jacobian.

**Reply by RSuryaNarayan** [reply]
thanks a lot for the info @drummerdoc @baperry2 @marchdf. I will keep you posted on what we end up doing for this!
